src/main/java/org/launhcode/healthynutrition/models/service/Utility.java

```diff
@@ -0,0 +1,10 @@
+package org.launhcode.healthynutrition.models.service;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class Utility {
```

Comment:
Creating a class for this function seems like overkill, just add the method to the class its used in